### ISSUE TITLE
Can a server target a single client with an RPC?

### ISSUE THREAD

**Issue by octavariain** [chkr1011/MQTTnet#572]
We have an issue where multiple clients (not under our control) subscribe to the topic:

`v1/devices/me/rpc/request/+`

However, we would like to target an individual client with RPCs from the server. So, for example, we would like our server to publish a message to:

`v1/devices/me/rpc/request/12345`

but target an individual client. Is this even possible?

**Comment by JanEggers** [comment]
depends on if your client can figure out /me/

**Comment by chkr1011** [comment]
You must add some "unique" thing to your topics like client IDs tokens, or similar. Otherwise you have no chance to "contact" a special client.

**Comment by octavariain** [comment]
We found a workaround to identify a single client - using the IMqttServer.Options.ClientMessageQueueInterceptor was the answer.

It appears that this action is fired by the broker just before publishing a method to a subscribed client. There is an AcceptEnqueue property that, if set to false, will not send the message, so we added logic to only set AcceptEnqueue to true if a specific condition was met - in our case the condition allowed us to identify an individual client.

Not sure if this is the way it was meant to be used, but it solved our problem!
